saleor/order/actions.py

```diff
@@ -168,22 +169,70 @@ def cancel_order(
     send_order_canceled_confirmation(order, user, app, manager)
 
 
+def make_refund(order, payments, info):
+    not_refunded_payments = []
+
+    for item in payments:
+        payment = item["payment"]
+        amount = item["amount"]
+        transaction = try_payment_action(
```

Comment:
I would propose to build a separate function for payment refund. Currently, we have try_payment_action which is a generic function and it complicates any customization for single action (like capture). 
Here is the PR https://github.com/mirumee/saleor/pull/8061. The assumption of converting `try_payment_action` into separate functions was defined in SALEOR-3782

Reply:
Also - `try_payment_action` in current form raises an error. Which will break a refund action.

Reply:
Ok, so I've created a new action for this, called `try_refund`. Let's keep `try_payment_action` until it isn't used anywhere and then remove it from the codebase.

Reply:
Great! 👍